Aztec/Classes/Libxml2/DOMString.swift

```diff
@@ -435,6 +448,23 @@ extension Libxml2 {
             rootNode.replaceCharacters(inRange: range, withElement: elementDescriptor)
         }
         
+        private static let headerLevels: [StandardElementType] = [.h1, .h2, .h4, .h5, .h6]
```

Comment:
We're placing constants at the top of the files mostly, so it would be good to to the same for this one.